kolibri/plugins/demo_server/assets/src/DemoServerBannerContent.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+
+  <h1 v-if="bannerClosed" class="banner-header">
+    {{ $tr('demoServerHeader') }}
+  </h1>
+  <div v-else>
+    <h1>{{ $tr('demoServerHeader') }}</h1>
```

Comment:
with the new implementation, this can be simplified to just putting the `v-if` on the `<h1>`!

Reply:
(actually not quite - nevermind)